velero-2-mixin/panels.libsonnet

```diff
@@ -0,0 +1,95 @@
+local utils = commonlib.utils;
+{
+  new(this):
+    {
+      local t = this.grafana.targets,
+      local stat = g.panel.stat,
+      local alertList = g.panel.alertList,
+
+      // create stat panel using commonlib
+      successfulBackupsCount:
+        commonlib.panels.generic.stat.info.new(
+          'Successful backups',
+          targets=[t.succesfulBackups],
+          description='Number of successful backups'
+        )
+        + stat.options.withGraphMode('area'),
+
+      failedBackups:
+        commonlib.panels.generic.stat.info.new(
+          'Failed backups',
+          targets=[t.failedBackups],
+          description='Number of failed backups.'
+        )
+        + stat.options.withGraphMode('area'),
+
+      successfulRestores:
+        commonlib.panels.generic.stat.info.new(
+          'Succesful restores',
+          targets=[t.succesfulRestores],
+          description='Number of succesful restores.'
+        )
+        + stat.options.withGraphMode('area'),
+      failedRestores:
+        commonlib.panels.generic.stat.info.new(
+          'Failed restores',
+          targets=[t.failedRestores],
+          description='Number of failed restores'
+        )
+        + stat.options.withGraphMode('area'),
+
+      alertsPanel:
+        alertList.new('PgBouncer alerts')
+        + alertList.options.UnifiedAlertListOptions.withAlertInstanceLabelFilter(this.grafana.variables.queriesGroupSelectorAdvanced),
+
+      topClustersByBackup:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top clusters by backups',
+          targets=[t.topClustersByBackupSuccess, t.topClustersByBackupFailure],
+          description=|||
+            The top clusters by number of backups.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ops'),
+
+      topClustersByRestore:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top clusters by restores',
+          targets=[t.topClustersByRestoreSuccess, t.topClustersByRestoreFailure],
+          description=|||
+            Top clusters by number of restores.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ops'),
+
+      topClustersByBackupSize:
+        commonlib.panels.memory.timeSeries.base.new(
+          'Top clusters by backup size',
+          targets=[t.topClustersByBackupSize],
+          description=|||
+            Top clusters by size of backups.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ops'),
+
+      topClustersByVolumeSnapshots:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top clusters by volume snapshots',
+          targets=[t.topClustersByVolumeSnapshotSuccess, t.topClustersByVolumeSnapshotFailure],
+          description=|||
+            Top clusters by number of volume snapshots.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ops'),
```

Comment:
Is `ops` the correct unit here? Seems like all units are `ops`, so I'd double check your units

Reply:
I think there restores/backups/snapshots can all be considered ops right?

Reply:
If it has no obvious unit or is a count, usually I put `` or `none`. Usually I reserve ops for when specific operations are being measured